auth0-js/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Adrian Chia <https://github.com/adrianchia>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace auth0 {
+declare module auth0 {
```

Comment:
Can you link to a simplified repro (ie: entire repo I could clone)? This sounds like it might be an issue we need to fix elsewhere.